templates/contrib/pc/rhel6-server-large-pc-i440fx.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: rhel6-server-large-pc-i440fx
+  annotations:
+    openshift.io/display-name: "Red Hat Enterprise Linux 6.0+ VM with i440fx machine type"
+    description: >-
+      Template for Red Hat Enterprise Linux 6 VM or newer.
+      A PVC with the RHEL disk image must be available.
+    tags: "hidden,kubevirt,virtualmachine,linux,rhel"
+    iconClass: "icon-rhel"
+    openshift.io/provider-display-name: "KubeVirt"
+    openshift.io/documentation-url: "https://github.com/kubevirt/common-templates"
+    openshift.io/support-url: "https://github.com/kubevirt/common-templates/issues"
+    template.openshift.io/bindable: "false"
+    template.kubevirt.io/version: v1alpha1
+    defaults.template.kubevirt.io/disk: rootdisk
+    template.kubevirt.io/editable: |
+      /objects[0].spec.template.spec.domain.cpu.sockets
+      /objects[0].spec.template.spec.domain.cpu.cores
+      /objects[0].spec.template.spec.domain.cpu.threads
+      /objects[0].spec.template.spec.domain.resources.requests.memory
+      /objects[0].spec.template.spec.domain.devices.disks
+      /objects[0].spec.template.spec.volumes
+      /objects[0].spec.template.spec.networks
+    name.os.template.kubevirt.io/rhel6.0: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.1: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.10: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.2: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.3: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.4: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.5: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.6: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.7: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.8: Red Hat Enterprise Linux 6.0 or higher
+    name.os.template.kubevirt.io/rhel6.9: Red Hat Enterprise Linux 6.0 or higher
+  labels:
+    os.template.kubevirt.io/rhel6.0: "true"
+    os.template.kubevirt.io/rhel6.1: "true"
+    os.template.kubevirt.io/rhel6.10: "true"
+    os.template.kubevirt.io/rhel6.2: "true"
+    os.template.kubevirt.io/rhel6.3: "true"
+    os.template.kubevirt.io/rhel6.4: "true"
+    os.template.kubevirt.io/rhel6.5: "true"
+    os.template.kubevirt.io/rhel6.6: "true"
+    os.template.kubevirt.io/rhel6.7: "true"
+    os.template.kubevirt.io/rhel6.8: "true"
+    os.template.kubevirt.io/rhel6.9: "true"
+    workload.template.kubevirt.io/server-pc-i440fx: "true"
+    flavor.template.kubevirt.io/large: "true"
+    template.kubevirt.io/type: "base"
+    template.kubevirt.io/version: "devel"
+objects:
+- apiVersion: kubevirt.io/v1
```

Comment:
Does this match the expected API version?

Reply:
On current downstream kubevirt-hyperconverged-operator.v2.6.5 I got
```
[dominik@t460p pc]$ oc process --local -f rhel6-server-large-pc-i440fx.yaml | oc apply -f -
unable to recognize no matches for kind "VirtualMachine" in version "kubevirt.io/v1"
```

Reply:
IIUC, this version should be fine.
It's also the version that appears in all templates in this repo.
For example: https://github.com/kubevirt/common-templates/blob/master/templates/rhel6.tpl.yaml#L42

Reply:
I think `v1` virtual machines were introduced in CNV 4.8

Reply:
The `v1` version is available since kubevirt `v0.37.0`. 
If we want the template to be usable on older kubevirt, you can simply change the apiVersion:
```
apiVersion: kubevirt.io/v1alpha3
```
The template should be compatible with the older version without any other changes.

Reply:
somehow I was sure that we've introduced it in 0.36 and it's part of the 2.6 release.
But you are right, thank you for point this out.
I'll change it to `kubevirt.io/v1alpha3`

Reply:
Thanks, works now on kubevirt-hyperconverged-operator.v2.6.5 . I don't have enough knowledge to understand the permission model of the dataVolumeTemplates source, but it might be fine.